backend/server.py

```diff
@@ -14,8 +19,16 @@ def home():
 def prescription_input():
     prescription = request.get_json()
     print(prescription)
+    if float(prescription["SPH_OD"]) == 0.0 or float(prescription["SPH_OS"]) == 0.0:
```

Comment:
Ideally, we should be able to handle an empty prescription. Does the generator method not support that atm?

Reply:
At the moment, it doesn't allow it:
![image](https://github.com/Lenscrafters-Capstone/LensCrafters-free/assets/15898988/1ae7f7bf-d8c6-4051-aee6-c840c9d1f4fe)

I do agree we should allow an empty prescription, will open an issue for it.

Reply:
What do you expect the behavior of empty prescription to be? Could make it so a prescription of 0 is converted to a radius of like 10,000 or 100,000 to approximate a flat lens.